conf/nextpnr/requirements.txt

```diff
@@ -1,10 +1,10 @@
 # Can be removed after merging https://github.com/SymbiFlow/prjxray/pull/1550
 pyyaml
+fasm
 git+git://github.com/SymbiFlow/prjxray.git#egg=prjxray
-git+git://github.com/SymbiFlow/fasm.git#egg=fasm
-git+git://github.com/SymbiFlow/symbiflow-rr-graph.git#egg=rr-graph
+git+git://github.com/SymbiFlow/symbiflow-rr-graph.git@951e72f0af1434b785b6617fc8979b413a9909fe#egg=rr-graph
```

Comment:
Change these from `git+git` to `git+https` (also for all the other instances)

Reply:
@acomodi Done, CI is green now.